src/UpdateWizard.js

```diff
@@ -265,7 +264,7 @@ class UpdateWizard extends InstallWizard {
           play.status = '';
         }
       });
-      this.updateGlobalState('playbookStatus', playStatus);
+      await  this.updateGlobalState('playbookStatus', playStatus);
```

Comment:
nitpick, double spaces here... might need to lint this

Reply:
@jeremy-moffitt  ok will fix